tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -587,6 +587,92 @@ def first_location_of_minimum(x):
     return np.argmin(x) / len(x) if len(x) > 0 else np.NaN
 
 
+@set_property("fctype", "aggregate")
+@not_apply_to_raw_numbers
+def percentage_of_doubled_datapoints(x):
+    """
+    Returns the percentage of unique value, that are present in the dataset
+    more than once.
+
+        len(different values occurring more than once) / len(different values)
+
+    This means the percentage is normalized to the number of unique values,
+    in contrast to the ratio_of_doubled_values.
+
+    :param x: the time series to calculate the feature of
+    :type x: pandas.Series
+    :return: the value of this feature
+    :return type: float
+    """
+    x = pd.Series(x)
+    return (x.value_counts() > 1).mean()
+
+
+@set_property("fctype", "aggregate")
+@not_apply_to_raw_numbers
+def ratio_of_doubled_values(x):
+    """
+    Returns the ratio of unique value, that are present in the dataset
+    more than once.
+
+        # of rows occuring more than once / # of all rows
+
+    This means the ratio is normalized to the number of rows in the dataset,
+    in contrast to the percentage_of_doubled_datapoints.
+
+    :param x: the time series to calculate the feature of
+    :type x: pandas.Series
+    :return: the value of this feature
+    :return type: float
+    """
+    x = pd.Series(x)
+
+    if len(x) == 0:
+        return np.nan
+
+    value_counts = x.value_counts()
+    return 1.0 * value_counts[value_counts > 1].sum() / len(x)
+
+
+@set_property("fctype", "aggregate")
+@not_apply_to_raw_numbers
+def sum_of_doubled_values(x):
```

Comment:
This is however not the number, but really  the sum of those values (in line 652, you see it is the weigted sum of all those reoccurring datapoints). But sum_of_reocurring_values is good as well.